tests/e2e/run-compliance-e2e.sh

```diff
@@ -24,22 +24,26 @@ test_compliance_e2e() {
 
     export_test_environment
 
+    export SENSOR_HELM_DEPLOY=true
+
     setup_deployment_env false false
     remove_existing_stackrox_resources
+    remove_compliance_operator_resources
     setup_default_TLS_certs
 
+    # Compliance v2 requires the Compliance Operator to be installed.
+    install_the_compliance_operator
     deploy_stackrox
-    # This is what installs the Compliance Operator if the
-    # INSTALL_COMPLIANCE_OPERATOR environment variable is set to "true".
-    deploy_optional_e2e_components
+
+    rm -f FAIL
```

Comment:
probably should put a comment on this.  Also maybe move it above where we cleanup existing resources?

Reply:
Aha - yes. Since this technique also exists in other e2e scripts, can I do that in a separate patch to apply it to all of them?

Reply:
I'd say just move it down to `run_compliance_e2e_tests()`. i.e. where it is used.